build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+echo "Starting the build process..."
+./gradlew build
+
+echo "Build completed. Copying the generated JavaScript file to the React app's public directory..."
+new_file=../Safehill-WebUI/public/kmp-shared.js
+cp shared/build/dist/js/productionExecutable/shared.js $new_file
+
```

Comment:
If you want, you could configure the output file path following the [link](https://kotlinlang.org/docs/js-project-setup.html#distribution-target-directory).
I guess the path would look something like:
"${projectDir.parentFile.parentFile}/Safehill-WebUI/public"
(Haven't tested it)